public/index.html

```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="de">
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
-    <title>Document</title>
+    <title>Capstone Project of alicie</title>
```

Comment:
I think there is a typo "Alice" or? :-)

Reply:
Well paid attention! <3 
This is on purpose, alicie = short for **Ali**ce **Cie**slik, like my GitHub username. It's only a working title here for the moment :P

Reply:
Ah! That makes sense. :D